svgReact/icons/__tests__/getColorVariables.test.ts

```diff
@@ -8,7 +8,7 @@ test('BookmarkAdd', () => {
   const result = getColorVariables(fileContent);
   expect(result).toMatchInlineSnapshot(`
     "<svg width=\\"24\\" height=\\"24\\" viewBox=\\"0 0 24 24\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\">
-        <path d=\\"M8 2a4 4 0 0 0-4 4v14c0 1.684 1.953 2.615 3.26 1.554l4.11-3.335a1 1 0 0 1 1.26 0l4.11 3.335c1.307 1.06 3.26.13 3.26-1.553V6a4 4 0 0 0-4-4H8Z\\" fill={secondaryColor || \\"#CACACE\\"} data-is-secondary-color=\\"true\\" />
+        <path d=\\"M8 2a4 4 0 0 0-4 4v14c0 1.684 1.953 2.615 3.26 1.554l4.11-3.335a1 1 0 0 1 1.26 0l4.11 3.335c1.307 1.06 3.26.13 3.26-1.553V6a4 4 0 0 0-4-4H8Z\\" fill={secondaryColor ? \`var(--amino-\${secondaryColor})\` : \\"#CACACE\\"} data-is-secondary-color=\\"true\\" />
```

Comment:
I'm guessing this test was broken because we aren't running our tests on build. I updated Vercel, so it should catch broken tests for next time.